cpp/src/slice2swift/Gen.cpp

```diff
@@ -1434,5 +1530,63 @@ Gen::ServantExtVisitor::visitInterfaceDefEnd(const InterfaceDefPtr&)
 void
 Gen::ServantExtVisitor::visitOperation(const OperationPtr& op)
 {
-    writeDispatchOperation(out, op);
```

Comment:
This is the only place `writeDispatchOperation` was called from, so I inlined it.
The code was copy-pasted from `SwiftUtil.cpp` with no changes.